packages/vue/src/utils/media-query.ts

```diff
@@ -49,11 +49,6 @@ function getScreenSizeInfo(): ScreenSizeInfo {
 
 export const screenInfo = ref(getScreenSizeInfo());
 
-export const isSmallScreen = () => {
```

Comment:
Why do you remove this?

Reply:
look at refactored form-popup. I found more simpler way